### ISSUE TITLE
Overflow Menu focus indicator is not visible in high-contrast mode

### ISSUE THREAD

**Issue by johnharan** [carbon-design-system/carbon#1460]
In high-contrast mode the focus indicator is not visible for the overflow menu button.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> What browser are you working in?

Firefox 60.2.1 esr

> What version of the Carbon Design System are you using?

"carbon-components": "9.48.0",
"carbon-components-angular": "2.0.8"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

## Steps to reproduce the issue

1. In windows os, open personalization, select a high contrast theme.
2. Open https://www.carbondesignsystem.com/components/data-table/code
3. Tab to the overflow menu button.
4. Note the focus indicator (square outline surrounding the button) is not visible.

## Additional information

![2018-11-14_12-20-13](https://user-images.githubusercontent.com/8806230/48621426-6811cd80-e99b-11e8-9dde-9817342221e2.png)
![2018-11-14_12-37-20](https://user-images.githubusercontent.com/8806230/48621427-6811cd80-e99b-11e8-8e0f-a9c54f8a6075.png)

Example of the focus indicator being visible in non-high-contrast-mode
![nonhcwithfocus](https://user-images.githubusercontent.com/8806230/48762961-1c656980-eca4-11e8-8851-830ae0b21e44.png)
